benchmarks/Elastic.Transport.Benchmarks/TransportBenchmarks.cs

```diff
@@ -0,0 +1,23 @@
+﻿using BenchmarkDotNet.Attributes;
+using Elastic.Transport.VirtualizedCluster;
+
+namespace Elastic.Transport.Benchmarks
+{
+	[MemoryDiagnoser]
+	public class TransportBenchmarks
```

Comment:
Maybe name this specifically to `VirtualizedClusterConnection` since that hides a whole state machine behind `client.Request()` it might not be reflective.

Probably best to inject an `InMemoryConnection` for baseline tests.

Reply:
I've fairly significantly changed the setup in the latest commit, which may be better, but equally may need more changes!